ntbs-service/Pages/Notifications/NotificationModelBase.cs

```diff
@@ -31,6 +31,7 @@ public abstract class NotificationModelBase : PageModel, IWithNotificationBanner
             NotificationRepository = notificationRepository;
         }
         public int NumberOfLinkedNotifications { get; set; }
+        public int? LatestLinkedNotificationId { get; set; }

```

Comment:
For a little extra safety, you could make the setter private, since you only want callers of the class to be able to read this property.